.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
```

Comment:
This is the preset being added. You can see documentation for it here: [https://github.com/airbnb/javascript](https://github.com/airbnb/javascript)